server/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "babel-node ./bin/www",
     "dev": "cross-env NODE_ENV=dev nodemon --exec babel-node ./bin/www",
-    "test": "cross-env NODE_ENV=test ./node_modules/.bin/mocha --require @babel/register --exit --timeout 120000"
+    "test": "cross-env NODE_ENV=test ./node_modules/.bin/mocha"
```

Comment:
what does this change do? I remember we had the timeout since the tests took very long on my machine. Does this make it so it just runs the tests until it ends?

Reply:
I moved the options to a options file, which runs when you call mocha automatically:
https://github.com/hatchways/team-summer/pull/121/files#diff-afe515ca12c57a7d8090a2eedbc74f3a

Reply:
I just saw it, cool!